feat: add dinder-cartesi-dapp@optimism-goerli DApp

maxabcr2000 opened the pull request:
# Brief
Adding the dinder-cartesi DApp to be accepted in the infrastructure.

# Main Activities
Added .yaml file as requested in the documentation.

endersonmaia commented on aetheras-dinder/optimism-goerli.yml:
```diff
@@ -0,0 +1,3 @@
+dapp:
+  image: gcr.io/aetheras-io/dinder-cartesi:v0.2.1
+  contractAddress: "0x69f54D30aC5b8aDdbE01F991210F55166BF0960d"
```
We'll need that flag to make it work on Optimism.

```suggestion
  contractAddress: "0x69f54D30aC5b8aDdbE01F991210F55166BF0960d"

validator:
  dispatcher:
    transactionManager:
      chainIsLegacy: "true"
```

maxabcr2000 commented:
> @maxabcr2000 please, give it a try on the `/graphsql` and `/inspect` paths for the following URLs.
> 
> * `https://0x69f54d30ac5b8addbe01f991210f55166bf0960d.optimism-goerli.rollups.staging.cartesi.io`
> * `https://aetheras-dinder.optimism-goerli.rollups.staging.cartesi.io`
> 
> I should approve and merge it after the changes I've asked for are applied and squashed into a single commit.

Fixed, please check.
